enum/abgabeart/abgabeart.go

```diff
@@ -0,0 +1,17 @@
+package abgabeart
+
+//go:generate stringer --type AbgabeArt
+//go:generate jsonenums --type AbgabeArt
+type AbgabeArt int
+
+const (
+	AbgabeArtKAS AbgabeArt = iota
+	AbgabeArtSA
+	AbgabeArtSAS
+	AbgabeArtTA
+	AbgabeArtTAS
+	AbgabeArtTK
+	AbgabeArtTKS
+	AbgabeArtTS
+	AbgabeArtTSS
```

Comment:
why the prefix `AbgabeArt` in front of all the enum members?
Is there any reason for that?